core-library/src/main/java/org/silverpeas/core/contribution/attachment/model/SimpleDocument.java

```diff
@@ -419,6 +425,21 @@ public long getWebdavContentEditionSize() {
     return webdavContentEditionSize;
   }
 
+  /**
+   * Gets the WEBDAV content description for the document if any.
+   * <p>
+   * No cache is handled here, it means that each time the method is called, each time webdav
+   * services are called.
+   * </p>
+   * @return an optional {@link WebdavContentDescriptor} instance.
+   */
+  public Optional<WebdavContentDescriptor> getWebdavContentDescriptor() {
```

Comment:
see my comment about WebdavContentDescriptor (this descriptor can describes any content that can be accessed or edited remotely whatever the protocol (WebDAV, WOPI, and later CMIS))